### PR TITLE
feat: Create report for pending Website Change Requests

### PR DESCRIPTION
**Pull request by andlar**

### CODE REVIEW

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    private Properties props;
+
+    @Autowired
+    private ChangeRequestDAO changeRequestDAO;
+
+    @Autowired
+    private CertificationBodyDAO certificationBodyDAO;
+
+    @Autowired
+    private Environment env;
```
Comment (Plow74) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java]: Could these be moved from field injection to Spring's preferred constructor injection?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+        } catch (MessagingException e) {
+            LOGGER.error(e);
+        } catch (EntityRetrievalException e) {
+            LOGGER.error(e);
+        }
+        LOGGER.info("********* Completed the Pending Change Request Email job. *********");
+    }
+
+    private List<List<String>> getAppropriateActivities(final JobExecutionContext jobContext,
```
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java]: Are you using the jobContext parameter in this method?

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+        currRow.set(DEVELOPER_CONTACT_NAME, activity.getDeveloper().getContact().getFullName());
+        currRow.set(DEVELOPER_CONTACT_EMAIL, activity.getDeveloper().getContact().getEmail());
+        currRow.set(DEVELOPER_CONTACT_PHONE_NUMBER, activity.getDeveloper().getContact().getPhoneNumber());
+        currRow.set(CHANGE_REQUEST_TYPE, activity.getChangeRequestType().getName());
+        currRow.set(CHANGE_REQUEST_STATUS, activity.getCurrentStatus().getChangeRequestStatusType().getName());
+        currRow.set(CHANGE_REQUEST_COMMENT, activity.getCurrentStatus().getComment());
+        currRow.set(CHANGE_REQUEST_DATE, Util.getTimestampFormatter().format(activity.getCurrentStatus().getStatusChangeDate()));
+
+        // Calculated time open
+        Date changeRequestDate = activity.getCurrentStatus().getStatusChangeDate();
```
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java]: Is the activity.getCurrentStatus().getStatusChangedDate() the date of the most recent status change? That's how I read it but maybe it's not. I would have thought you wanted to use the creation date of the change request here.

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java (hunk 1 above)
Comment (andlar) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java]: I'm actually not sure if they can. These are the jobs that are run via Quartz, and they all have weird environment things happening for them. I'll check though

Comment (andlar) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/PendingChangeRequestEmailJob.java]: Tried this @Plow74 , but the way the Quartz Jobs are created doesn't support the Autowired constructor thing for injection

**Pull request merged**